main.go

```diff
@@ -1,14 +1,23 @@
 package main
 
 import (
+	"log"
+
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
 
 	"github.com/DSSD-Madison/gmu/models"
 	"github.com/DSSD-Madison/gmu/routes"
+	"github.com/joho/godotenv"
```

Comment:
In golang, we usually organize imports like this:

system

external

internal

So this import should go with the other external ones above.